src/Rules/Drupal/Tests/BrowserTestBaseDefaultThemeRule.php

```diff
@@ -30,6 +30,26 @@ public function processNode(Node $node, Scope $scope): array
             return [];
         }
 
+        // Only inspect tests.
+        if (!str_ends_with($node->namespacedName->getLast(), 'Test')) {
+            return [];
+        }
```

Comment:
This is a PHP 8.0 function, so we'd need the old school `strpos` or something.

Reply:
its in `symfony/polyfill-php80` so it should be fine.

Reply:
nice! I never consider this

Reply:
well... i say that. its a indirect dependency so that's _technically_ not a good solution. replace it.